frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/SmartScalarSettingsWidgets.tsx

```diff
@@ -16,60 +19,109 @@ type SmartScalarComparisonWidgetProps = {
   value: SelectedComparison;
 };
 
+type Tab = "staticNumber" | null;
+
 export function SmartScalarComparisonWidget({
   onChange,
   options,
   value: selectedValue,
 }: SmartScalarComparisonWidgetProps) {
   const [open, setOpen] = useState(false);
+  const [tab, setTab] = useState<Tab>(getTabForValue(selectedValue));
```

Comment:
Are we sure that we want to open the menu straight to the `staticNumber` tab? It may be confusing for users to click on the comparison and then just see a back button (which in my mind, would make me assume I am just closing the menu). 

Can we ask product & design what they think is best?